metrics/examples/grafana/grafana.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
       - name: grafana
-        image: strimzilab/grafana-openshift:latest
+        image: grafana/grafana:latest
```

Comment:
I do not think we should use latest here. Can you fix it to some actual version which works and not to something what will be updated over the time?